mpas_analysis/shared/interpolation/interpolate.py

```diff
@@ -82,19 +83,28 @@ def build_remap_weights(sourceFileName, outWeightFileName,
         does nothing and returns immediately, potentially saving a costly
         re-computaiton of the mapping file.
 
+    Raises
+    ------
+    OSError
+        If `ESMF_RegridWeightGen` is not in the system path.
+
     Author
     ------
     Xylar Asay-Davis
 
     Last Modified
     -------------
-    02/25/2017
+    03/14/2017
     """
 
     if not overwrite and os.path.exists(outWeightFileName):
         # a valid weight file already exists, so nothing to do
         return
 
+    if find_executable('ESMF_RegridWeightGen') is None:
+        raise OSError('ESMF_RegridWeightGen not found. Make sure esmf package '
+                      'is installed via\nlatest nco: \n    conda install nco')
```

Comment:
sorry for not checking this earlier. The full command should be 'conda install -c conda-forge nco', as I tried conda install nco without doing anything else and that failed.